chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+
+    const fullName = self?.model?.trello?.user?.fullName || '';
+
+    self.popupView.bindData(self.model);
+
+    self.gmailView.parsingData = false;
+    self.model.gmail = self.gmailView.parseData({ fullName });
+    self.popupView.bindGmailData(self.model.gmail);
+  }
+
+  initialize() {
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)